### DISCUSSION TITLE
QueryFailedError: column AssetFaceEntity.embedding does not exist

### DISCUSSION THREAD

**Discussion by thehijacker** [immich-app/immich#7311] [Q&A]
I've managed to update to latest version and fix all the database issues. GUI now work. But for new images it not longer detects people. I get following error:

![image](https://github.com/immich-app/immich/assets/13857043/3bf1e7bc-abbe-44bf-8a18-3970407641bf)

Any idea why?

**Comment by thehijacker** [comment]
And when I try searching I get and error in GUI. In log I can see:

![image](https://github.com/immich-app/immich/assets/13857043/7bb2e5a7-4c40-44e1-8670-c6faf5e8e1b4)

**Reply by alextran1502** [reply]
Go to job and run "Smart Search" for all, Face Detection for all and Facial Recognition for all. It looks like you have dropped those tables?

**Reply by thehijacker** [reply]
Did not seem to help. It started job for over 120.000 images and my CPU peaked. But all I can see in log is:

![image](https://github.com/immich-app/immich/assets/13857043/e285d33e-044e-4d73-b15c-2bff12c16d30)

Under Explorer I can see the people. And when I select them it shows images only from them. So only new images are not recognnized.

**Comment by SitramSoft** [comment]
I have the same issue. Below is the error I get when I try to run smart search for all.
![image](https://github.com/immich-app/immich/assets/8177290/ecc2f1d4-f4b0-4fac-9c69-85db818a4830)

**Comment by timdonovanuk** [comment]
Same problem for me - how does one go about recreating these tables?
